bom/application/pom.xml

```diff
@@ -1612,6 +1612,16 @@
                 <artifactId>quarkus-websockets-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-websockets-client</artifactId>
```

Comment:
why websockets (plural) for the client and websocket (singular) for the server? 
I would go with singular everywhere, but maybe I'm missing the reason.